modules/serverless-integrationsource-creating-aws-credentials.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/event-sources/serverless-integrationsource.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-integrationsource-creating-aws-credentials_{context}"]
+= Creating AWS credentials
```

Comment:
I like just saying "Creating AWS credentials" at the top of the chapter

Reply:
I have added a single section at the top. Can you please verify?